### ISSUE TITLE
Cannot use setOnTouchListener

### ISSUE THREAD

**Issue by africantearoa** [chathuralakmal/AndroidImagePopup#11]
Using .setOnTouchListener() on the ImagePopup object does not seem to work - the the code within the OnTouchListener is never reached.

To try counter this, I tried manually pulling the files from this repository and creating my own custom ImagePopup class, adding the .setOnTouchListener() to the 'layout' field object. While that did execute the code correctly, the popup wouldn't go fullscreen so I abandoned whole custom class.
I'm a rookie at this, anyone know what I might be doing wrong?

**Comment by chathuralakmal** [comment]
@africantearoa  yes the code is not support for touchListener yet. and in default popup doesnt go full screen. you can see it in the popup window section how it has coded. anyway ill look on to it. hope you can come up with a great solution as well

**Comment by africantearoa** [comment]
I wonder if we could include a onTouchListener setter method for the layout
object in the popupImage class?
